omod/src/main/java/org/openmrs/module/aijar/htmlformentry/MCHProgramExitPostSubmissionAction.java

```diff
@@ -0,0 +1,56 @@
+package org.openmrs.module.aijar.htmlformentry;
+
+import java.util.Date;
+
+import org.openmrs.Patient;
+import org.openmrs.PatientProgram;
+import org.openmrs.Program;
+import org.openmrs.api.APIException;
+import org.openmrs.api.ProgramWorkflowService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.aijar.metadata.core.Programs;
+import org.openmrs.module.htmlformentry.CustomFormSubmissionAction;
+import org.openmrs.module.htmlformentry.FormEntryContext;
+import org.openmrs.module.htmlformentry.FormEntrySession;
+
+/**
+ * Exits patients out of the MCH program
+ */
+public class MCHProgramExitPostSubmissionAction implements CustomFormSubmissionAction {
```

Comment:
I do not seem to understand this well. Does it mean that the current classes do not allow future extension?